cron_main.py

```diff
@@ -63,5 +62,6 @@
                                ('/tasks/admin/clear_old_subs', AdminSubsClear),
                                ('/tasks/admin/enqueue/clear_old_webhooks', AdminWebhooksClearEnqueue),
                                ('/tasks/admin/clear_old_webhooks', AdminWebhooksClear),
+                               ('/tasks/admin/rebuild_district_teams/([0-9]+)', AdminCreateDistrictTeamsDo),
```

Comment:
should these be in the /math/ path? They mostly do math, vs administer user accounts.